final_project/Final_Project/README.md

```diff
@@ -0,0 +1,18 @@
+README.me
+================
+Tyler Allan
+3/8/2021
+
+### Final Project Folder
+
+This repository will hold all the files necessary for the final project
```

Comment:
Sounds interesting! I'm curious how you'll partial out the environmental influences in this dataset!

Reply:
I agree, interesting study! I think the use of twin data is to see if members within families have more similar interests than members between families to examine the heritability of interests?